src/OpenTK.GraphicsLibraryFramework/Interfaces/IGLFW.cs

```diff
@@ -1111,214 +1434,202 @@ internal interface IGLFW
         /// when rendering with OpenGL or OpenGL ES.
+        /// </para>
+        /// <para>
         /// If the swap interval is greater than zero,
         /// the GPU driver waits the specified number of screen updates before swapping the buffers.
-        ///
+        /// </para>
+        /// <para>
         /// The specified window must have an OpenGL or OpenGL ES context.
         /// Specifying a window without a context will generate a <see cref="ErrorCode.NoWindowContext"/> error.
-        ///
-        /// This function does not apply to Vulkan. If you are rendering with Vulkan, see vkQueuePresentKHR instead.
+        /// </para>
         /// </summary>
         /// <param name="window">The window whose buffers to swap.</param>
         /// <remarks>
+        /// <para>
         /// EGL: The context of the specified window must be current on the calling thread.
-        ///
+        /// </para>
+        /// <para>
         /// This function may be called from any thread.
-        ///
-        /// TODO: Possible errors include <see cref="ErrorCode.NotInitialized"/>, <see cref="ErrorCode.NoWindowContext"/> and <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
+        /// <para>
+        /// Possible errors include <see cref="ErrorCode.NotInitialized"/>, <see cref="ErrorCode.NoWindowContext"/> and <see cref="ErrorCode.PlatformError"/>.
+        /// </para>
         /// </remarks>
-        /// <seealso cref="SwapInterval"/>
         unsafe void SwapBuffers(Window* window);
 
         /// <summary>
+        /// <para>
         /// This function returns whether the specified API extension is supported
         /// by the current OpenGL or OpenGL ES context.
         /// It searches both for client API extension and context creation API extensions.
-        ///
+        /// </para>
+        /// <para>
         /// A context must be current on the calling thread.
         /// Calling this function without a current context will cause a <see cref="ErrorCode.NoWindowContext"/> error.
-        ///
+        /// </para>
+        /// <para>
         /// As this functions retrieves and searches one or more extension strings each call,
         /// it is recommended that you cache its results if it is going to be used frequently.
         /// The extension strings will not change during the lifetime of a context, so there is no danger in doing this.
-        ///
-        /// This function does not apply to Vulkan.
-        /// If you are using Vulkan, see glfwGetRequiredInstanceExtensions,
-        /// vkEnumerateInstanceExtensionProperties and vkEnumerateDeviceExtensionProperties instead.
+        /// </para>
         /// </summary>
         /// <param name="extensionName">The ASCII encoded name of the extension.</param>
         /// <returns><c>true</c> if the extension is available, or <c>false</c> otherwise.</returns>
         /// <remarks>
+        /// <para>
         /// This function may be called from any thread.
-        ///
```

Comment:
Why removed? Is available in 3.3.

Reply:
All Vulkan bindings/documentation was removed because our bindings can't support Vulkan at the moment, we lack several critical types (such as VkInstance). When OpenTK adds Vulkan support, this stuff will come back, but until then it's just confusing to have Vulkan docs/functions for a set of bindings that can't support it.